File: LeetCode/Easy/3110-score-of-a-string/3110-score-of-a-string.cpp
class Solution {
public:
    int scoreOfString(string s) {
        int ret = 0;
        char prev = s[0];
    
        for (int i = 1; i < s.length(); ++i){
            char temp = s[i];
            ret += abs(prev - temp);
            prev = temp;
        }
        

        return ret;
    }
};

File: LeetCode/Easy/3110-score-of-a-string/3110-score-of-a-string.py
class Solution:
    def scoreOfString(self, s: str) -> int:
        ret = 0
        prev = ord(s[0])
    
        for i in range(1, len(s)):
            temp = ord(s[i])
            ret += abs(prev - temp)
            prev = temp

        return ret